pnpm-lock.yaml

```diff
@@ -1,96 +1,144 @@
-lockfileVersion: 5.4
-
-specifiers:
-  '@logux/eslint-config': ^48.0.0
-  autoprefixer: ^10.4.13
-  browserslist: ^4.21.4
-  css-tree: ^2.3.0
-  cssom: ^0.5.0
-  eslint: ^8.29.0
-  eslint-config-standard: ^17.0.0
-  eslint-plugin-import: ^2.26.0
-  eslint-plugin-n: ^15.6.0
-  eslint-plugin-prefer-let: ^3.0.1
-  eslint-plugin-promise: ^6.1.1
-  gonzales: ^1.0.7
-  gulp: ^4.0.2
-  gulp-bench: ^1.1.0
-  gulp-bench-summary: ^0.1.0
-  less: ^4.1.3
-  lightningcss: ^1.17.1
-  mensch: ^0.3.4
-  node-sass: ^8.0.0
-  parserlib: ^1.1.1
-  postcss: ^8.4.20
-  postcss-mixins: ^9.0.4
-  postcss-nested: ^6.0.0
-  postcss-selector-parser: ^6.0.11
-  postcss-simple-vars: ^7.0.1
-  postcss-value-parser: ^4.2.0
-  rework: ^1.0.1
-  sass: ^1.56.2
-  stylecow-core: ^2.5.0
-  stylecow-plugin-calc: ^2.0.0
-  stylecow-plugin-nested-rules: ^5.0.1
-  stylecow-plugin-prefixes: ^6.0.5
-  stylecow-plugin-variables: ^5.1.4
-  stylis: ^4.1.3
-  through2: ^4.0.2
+lockfileVersion: '6.0'
```

Comment:
This was unintentional.

Is it ok that this changes?
Or should I install a different version of `pnpm` so that this remains constant?

Reply:
Don't worry, I will fix it on my own